### DISCUSSION TITLE
Thank you so much for this awesome tool! Rector Team!

### DISCUSSION THREAD

**Discussion by alexandreelise** [rectorphp/rector#8901] [General]
Thanks to rector, I saved hours of grunt work. Soul taking tedious task. Rector did it flawlessly in a couple of minutes. Imagine having to check manually if there is a mistake somewhere in a million- line codebase. Thanks Super Rector Team. Have a delightful day!

**Comment by TomasVotruba** [comment]
Awesome to read this Alex! I wish you'll create amazing stuff with the saved time :rocket: 
Our pleasure  😊 :pray:
